### DISCUSSION TITLE
Incorrect sort double field

### DISCUSSION THREAD

**Discussion by MohMehrnia** [ravendb/ravendb#19380] [General]
I used Below map/reduce

```
//map
from v in docs.MaxCurrentViewByUserIDAndFileID
select new {
    v.UserID, 
    v.FullName, 
    TotalViewTime = v.CurrentTime
}

//reduce
from r in results
group r by new { r.UserID, r.FullName } into g
select new {
   g.Key.UserID,
   g.Key.FullName,
   TotalViewTime = g.Sum(x=>x.TotalViewTime)
}
```

when I run below query
`from index 'TopUser' order by TotalViewTime desc `

result not correct sorted
[SIN-PLATFORM_A_query_result_20241005_1643.csv](https://github.com/user-attachments/files/17266859/SIN-PLATFORM_A_query_result_20241005_1643.csv)

**Comment by krivulcik** [comment]
Try adding `as double` do your query, like this:

`from index 'TopUser' order by TotalViewTime as double desc`

That way, RavenDB will know that the value is numeric and will sort the values correctly.

See https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#sorting-query-results for further details.
